special-order-numbers.md

```diff
@@ -1,3 +1,11 @@
 # Create a Special Order for Telephone Numbers
 
-Details on how to place special orders for telephone numbers will be available soon.
+Customers who are interested in a special order (ex. acquiring large quantities of numbers) can do so by following the process below. 
+
+*PLEASE NOTE:* You need to have an eligible Azure Subscription and Resource in the US (see link here for eligibility criteria) to be able to purchase numbers. 
+
+You will need to work through the following steps: 
+- Download this form: [Manual / Bulk Number Acquisition Form]()
```

Comment:
Any way we can provide a web-based form link to make it easier for folks?

Reply:
Question for Sabya, my understanding is that due to privacy concerns we need to do it this way to comply with support desk.

Reply:
@dassabya15

Reply:
That's correct - we'll need to follow the official guidelines here @ddematheu2 , @mikben

Reply:
Also, could we add the link to the pdf / word file you helped create?

Reply:
Will add once I push the file live. I need to do a quick rename on it before adding here.